MiniApp/en.lproj/Localizable.strings

```diff
@@ -37,3 +37,8 @@
 "wait_message" = "Please wait...";
 "ok" = "OK";
 "cancel" = "Cancel";
+
+// MARK: - UI
+"ui_close_button_title" = "Close";
```

Comment:
Are these the key names usable by the Host App to override the text values? Shouldn't they be prefixed with something specific to mini apps? i.e. `miniapp_ui_close_button_title`